api/pyproject.toml

```diff
@@ -0,0 +1,17 @@
+[project]
+requires-python = ">=3.10"
+
+[tool.ruff]
+exclude = [
+    "__init__.py",
+    "tests/",
+]
+line-length = 120
+
+[tool.ruff.lint]
+ignore-init-module-imports = true
+select = [
+    "F401", # unused-import
+    "I001", # unsorted-imports
+    "I002", # missing-required-import
+]
```

Comment:
Three rules are enabled by demand. And the list is extendable for other linter rules of Flake8, pycodestyle and etc.   (refer to https://docs.astral.sh/ruff/rules)